Description: Tell libtool not to deduplicate linked libraries which causes problems in the case of circular deps. Force -lc to be added at the end.
Author: Iain Lane <iain.lane@canonical.com>

Index: gst-plugins-bad1.0/ext/modplug/Makefile.am
===================================================================
--- gst-plugins-bad1.0.orig/ext/modplug/Makefile.am
+++ gst-plugins-bad1.0/ext/modplug/Makefile.am
@@ -2,7 +2,7 @@ plugin_LTLIBRARIES = libgstmodplug.la
 
 libgstmodplug_la_SOURCES = gstmodplug.cc
 libgstmodplug_la_CXXFLAGS = $(GST_PLUGINS_BASE_CFLAGS) $(GST_CXXFLAGS) $(MODPLUG_CFLAGS)
-libgstmodplug_la_LIBADD = $(GST_PLUGINS_BASE_LIBS) $(LIBM) $(MODPLUG_LIBS)
-libgstmodplug_la_LDFLAGS = $(GST_PLUGIN_LDFLAGS)
+libgstmodplug_la_LIBADD = $(GST_PLUGINS_BASE_LIBS) $(LIBM) $(MODPLUG_LIBS) -lc
+libgstmodplug_la_LDFLAGS = $(GST_PLUGIN_LDFLAGS) --preserve-dup-deps
 
 noinst_HEADERS = gstmodplug.h
